infrastructure/docker/build/docker-compose.yml

```diff
@@ -117,6 +117,23 @@ services:
       - ../../..:/trafficcontrol:z
       - ../../../.npm:/root/.npm:z
 
+  traffic-portal_build:
```

Comment:
The build log is named `build-experimental-traffic-portal.log`. I'm not particular about what name is chosen, but the service and log should match.